krux_marathon_api/marathonapi.py

```diff
@@ -71,8 +52,12 @@ def connect(self, address, port):
             s.close()
```

Comment:
The exception can be helpful in debugging issues though, why would I get rid of it?